frameworks/PHP/fat-free/setup.sh

```diff
@@ -1,12 +1,20 @@
 #!/bin/bash
 
-fw_depends php nginx
+fw_depends php7 nginx
 
 sed -i 's|localhost|'"${DBHOST}"'|g' index.php
-sed -i 's|.*/FrameworkBenchmarks/php-fatfree|'"${TROOT}"'|g' deploy/php
-sed -i 's|Directory .*/FrameworkBenchmarks/php-fatfree|Directory '"${TROOT}"'|g' deploy/php
-sed -i 's|root .*/FrameworkBenchmarks/php-fatfree|root '"${TROOT}"'|g' deploy/nginx.conf
+sed -i 's|root .*/FrameworkBenchmarks/fat-free|root '"${TROOT}"'|g' deploy/nginx.conf
 sed -i 's|/usr/local/nginx/|'"${IROOT}"'/nginx/|g' deploy/nginx.conf
 
+
+F3DIR="$TROOT/src"
+
+[[ ! -e "$F3DIR" ]] || rm -r "$F3DIR"
```

Comment:
The `rm` command here needs the `f` flag to force the `rm` to happen without bringing up a prompt, which was hanging Travis and my local install due to prompting about write-protected files.